src/scripts/main.js

```diff
@@ -1,3 +1,53 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.onclick = () => {
+    resolve();
+  };
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(reject, 3000);
+});
+
+promise1.then(
+  function (value) {
+    myDisplayer(value);
+  },
+  function (error) {
+    myDisplayer2(error);
+  },
```

Comment:
The `then` method of a promise takes two arguments: a callback for when the promise is resolved and a callback for when it is rejected. The third argument here is unnecessary and should be removed.